omnisend/class-omnisend-core-bootstrap.php

```diff
@@ -129,26 +179,52 @@ public static function cron_schedules( $schedules ) {
 		return $schedules;
 	}
 
-	public static function load_omnisend_admin_styles(): void {
-		// phpcs:disable WordPress.Security.NonceVerification
-		if ( isset( $_GET['page'] ) ) {
-			if ( in_array( $_GET['page'], array( 'omnisend', 'omnisend-app-market' ), true ) ) {
-				wp_enqueue_style(
-					'roboto.css',
-					plugin_dir_url( __FILE__ ) . 'assets/fonts/roboto/roboto.css?' . time(),
-					array(),
-					OMNISEND_CORE_PLUGIN_VERSION,
-				);
-				wp_enqueue_style(
-					'styles.css',
-					plugin_dir_url( __FILE__ ) . 'styles/styles.css?' . time(),
-					array(),
-					OMNISEND_CORE_PLUGIN_VERSION,
-				);
-			}
+
+	private static function migrate_options() {
+		$is_store_connected = Options::is_store_connected();
+		$visit_count        = Options::get_landing_page_visit_count();
+
+		$store_connected_before_visit_count_option = $is_store_connected && $visit_count === 0;
+		if ( $store_connected_before_visit_count_option ) {
+			Options::set_landing_page_visited();
 		}
 	}
 
+
+	private static function show_notification_icon(): bool {
+		return ! Options::is_landing_page_visited();
+	}
+
+	public static function load_omnisend_admin_styles(): void {
+		wp_enqueue_style(
+			'roboto.css',
+			plugin_dir_url( __FILE__ ) . 'assets/fonts/roboto/roboto.css?' . time(),
+			array(),
+			OMNISEND_CORE_PLUGIN_VERSION,
+		);
+		wp_enqueue_style(
+			'styles.css',
+			plugin_dir_url( __FILE__ ) . 'styles/styles.css?' . time(),
+			array(),
+			OMNISEND_CORE_PLUGIN_VERSION,
+		);
+		wp_enqueue_style(
+			'site-styles.css',
+			plugin_dir_url( __FILE__ ) . 'styles/site-styles.css?' . time(),
+			array(),
+			OMNISEND_CORE_PLUGIN_VERSION,
+		);
+	}
+
+	public static function load_omnisend_site_styles(): void {
+		wp_enqueue_style(
+			'site-styles.css',
+			plugin_dir_url( __FILE__ ) . 'styles/site-styles.css?' . time(),
```

Comment:
Why /site-styles inserted twice?

Reply:
one time for admin, one time for website styles ( when wp admin bar is visible in admin)